care/emr/reports/renderer/template_engine.py

```
@@ -0,0 +1,164 @@
+        except Exception as e:
+            return False, f"Template validation error: {e!s}"
+
+    def render(self, template_string: str, context: dict) -> str:
+        try:
+            template = self.env.from_string(template_string)
+            return template.render(**context)
+        except TemplateSyntaxError as e:
+            msg = f"Template syntax error at line {e.lineno}: {e.message}"
+            raise TemplateSyntaxError(msg, e.lineno) from e
+        except Exception as e:
+            raise e
```

Ruff (0.14.6)

Use `raise` without specifying exception name

Remove exception name

(TRY201)